Lib/test/test_curses.py

```diff
@@ -1095,6 +1095,16 @@ def test_resizeterm(self):
         self.assertEqual(curses.LINES, lines)
         self.assertEqual(curses.COLS, cols)
 
+        for func in (curses.resizeterm, curses.resize_term):
+            with self.subTest(func=func):
+                with self.assertRaises(OverflowError):
+                    curses.resizeterm(35000, 1)
+                with self.assertRaises(OverflowError):
+                    curses.resizeterm(1, 35000)
```

Comment:
```suggestion
                with self.assertRaises(OverflowError):
                    func(35000, 1)
                with self.assertRaises(OverflowError):
                    func(1, 35000)
```

That's maybe why you didn't spot the failure?